### DISCUSSION TITLE
using the meta data in the template but it can't work

### DISCUSSION THREAD

**Discussion by moorebi** [BryanJenksCommunity/FAQ#110] [Q&A] [answered]
I am getting the same issue here when I try to use Templater to insert the 'meta' template on a blank file called `{ book`

- Templater enabled with following settings.. I have tried putting in the folder `Templates` and also leaving blank and does not make difference
![image](https://user-images.githubusercontent.com/6247939/130463020-81c9ced7-e904-4c9d-b4fd-865cbe79f937.png)

![image](https://user-images.githubusercontent.com/6247939/130463135-95772ef7-85df-47de-8f45-5bcbbd3449a7.png)

- The `meta` file is setup from your .zip download
![image](https://user-images.githubusercontent.com/6247939/130463598-ccea37cb-047d-4e1c-b171-86a2946862ad.png)

### Discussed in https://github.com/BryanJenksCommunity/FAQ/discussions/85

<div type='discussions-op-text'>

<sup>Originally posted by **fred1357944** July  5, 2021</sup>
<img width="694" alt="image" src="https://user-images.githubusercontent.com/34299763/124428900-ccc60480-dd9f-11eb-8e8a-1b6ce4c63e3f.png">

when I import the meta template into my new file named "{test" but it shows up this announcement. Does anyone know how to fix this?

Thank you so much...</div>

[a comment by an automated account is not shown]

**Comment by tallguyjenks** [comment, marked as answer]
in #85 i think we found that the the template file names need to be notes. specifically [this reply](https://github.com/BryanJenksCommunity/FAQ/discussions/85#discussioncomment-1147860)
